core/src/main/java/org/elasticsearch/index/query/FilteredQueryParser.java

```diff
@@ -83,44 +78,35 @@ public Query parse(QueryParseContext parseContext) throws IOException, QueryPars
             }
         }
 
-        // parsed internally, but returned null during parsing...
         if (query == null) {
-            return null;
+            /*
+             * What the hack happened here? What we got from the user looks something like this:
+             *
+             * {
+             *   "query": {
+             *      "filtered": {
+             *          "query": { },
+             *          "filter": {"range": { "created": { "gte": "now - 1d / d" }}}
+             *      }
+             *   }
+             * }
+             * */
+            return EmptyQueryBuilder.PROTOTYPE;
```

Comment:
Agree with this, I already left a comment about this further up the timeline (https://github.com/MaineC/elasticsearch/commit/fd127841d6c907980b5b0328422347e50e855ee7#commitcomment-12034514)